frontend/packages/operator-lifecycle-manager/src/components/index.tsx

```diff
@@ -114,7 +114,11 @@ export const providedAPIForModel = (csv: ClusterServiceVersionKind, model: K8sKi
 
 export const parseALMExamples = (csv: ClusterServiceVersionKind) => {
   try {
-    return JSON.parse(csv?.metadata?.annotations?.['alm-examples'] ?? '[]');
+    return JSON.parse(
+      csv?.metadata?.annotations?.['operatorframework.io/initialization-resource'] ??
```

Comment:
The initialization resource is just a single object, right, not an array? This would return a different type.

Reply:
Correct, I used an array in the init resource during my test data which is incorrect.  I have fixed that and the code.